src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.StaticWebAssets.targets

```diff
@@ -0,0 +1,172 @@
+<!--
+***********************************************************************************************
+Microsoft.NET.Sdk.Razor.StaticWebAssets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved.
+***********************************************************************************************
+-->
+
+<Project ToolsVersion="14.0">
+
+  <!-- Targets that support static content scenarios in ASP.NET Core.
+       The main targets are:
+       * GenerateStaticWebAssetsManifest: Creates a manifest file to use in development with
+         the paths to all the references packages and projects content roots.
+       * ResolveStaticWebAssetsInputs: Collects all the static assets from different sources
+         * Current project.
+         * Referenced project.
+         * Referenced packages.
+  -->
+
+  <UsingTask
+    TaskName="Microsoft.AspNetCore.Razor.Tasks.GenerateStaticWebAssetsManifest"
+    AssemblyFile="$(RazorSdkBuildTasksAssembly)"
+    Condition="'$(RazorSdkBuildTasksAssembly)' != ''" />
+
+  <PropertyGroup>
+    <GenerateStaticWebAssetsManifestDependsOn>
+      ResolveStaticWebAssetsInputs;
+      _CreateStaticWebAssetsInputsCacheFile
+    </GenerateStaticWebAssetsManifestDependsOn>
+
+    <AssignTargetPathsDependsOn>
+      GenerateStaticWebAssetsManifest;
+      $(AssignTargetPathsDependsOn)
+    </AssignTargetPathsDependsOn>
+
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <_GeneratedStaticWebAssetsInputsCacheFile>$(IntermediateOutputPath)Microsoft.AspNetCore.StaticWebAssets.cache</_GeneratedStaticWebAssetsInputsCacheFile>
```

Comment:
<_GeneratedStaticWebAssetsInputsCacheFile>$(IntermediateOutputPath).$(TargetName).Microsoft.AspNetCore.StaticWebAssets.cache

Also do you really want to namespace the file name? Really seems unnecessary (i.e. can we just call it `StaticWebAssets` here)? You can embed it using whatever name you want.

Reply:
I understand, I'm just picking a "unique" enough name

Reply:
Is there an actual problem with any of the two options that I chose here?
I don't understand why would I use the $(TargetName) or why would I pick a shorter name (other than path length constraints, but I don't think this is the file with the longest name in the folder anyway)

Reply:
Discussed offline. Updated